app/src/js/reducers/providers.js

```diff
@@ -62,7 +62,7 @@ export default function reducer (state = initialState, action) {
   switch (action.type) {
     case PROVIDER:
       set(state, ['map', id, 'inflight'], false);
-      set(state, ['map', id, 'data'], assignDate(data));
```

Comment:
These lines were what was adding the unnecessary `queriedAt` property to the rule/provider objects and causing them to fail when sent as the body of a PUT requests to the API, since `queriedAt` is not included in the schema for either of those objects. As far as I can tell and based on my testing, this `queriedAt` property wasn't providing anything, so I just removed it